docs/src/kernel/advanced-instructions.md

```diff
@@ -60,16 +60,31 @@ Then, in the interactive shell, run the following script to start the syscall te
 
 To debug Asterinas using [QEMU GDB remote debugging](https://qemu-project.gitlab.io/qemu/system/gdb.html), one could compile Asterinas in the debug profile, start an Asterinas instance and run the GDB interactive shell in another terminal.
 
-First, start a GDB-enabled VM of Asterinas and wait for debugging connection:
+Start a GDB-enabled VM of Asterinas with `GDB_SERVER=1` and wait for debugging connection:
 
 ```bash
 make run GDB_SERVER=1 ENABLE_KVM=0
 ```
 
-Next, start a GDB client in another terminal.
+Then debug via [a GDB client in terminal](#terminal), or [VS Code](#vs-code).
+
+Currently, the Asterinas runner's debugging interface is exposed via UNIX sockets. Thus there shouldn't be multiple debugging instances in the same container. To add debug symbols for the underlying infrastructures such as UEFI firmware or bootloader, please check the runner's source code for details.
+
+#### Terminal
+
+Start a GDB client with `GDB_CLIENT=1` in another terminal.
 
 ```bash
 make run GDB_CLIENT=1
 ```
 
-Currently, the Asterinas runner's debugging interface is exposed via UNIX sockets. Thus there shouldn't be multiple debugging instances in the same container. To add debug symbols for the underlying infrastructures such as UEFI firmware or bootloader, please check the runner's source code for details.
+#### VS Code
+
+[CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb) extension is required.
+
+After starting a debug server with `GDB_SERVER=1` from the shell as [described above](#using-gdb-to-debug), press `F5`(Run and Debug) to start a debug session via VS Code. 
+Now click `Continue`(or, press `F5`) at the fisrt break to resume the paused server instance, and it will continue until reaching your first breakpoint. 
+
+![GDB Debugging via VS Code](../images/gdb_debug_via_vscode.png)
```

Comment:
Binary files like PNG are not friendly to version control. My current strategy is (1) to be very restrained with adding image files into the Git history, (2) if adding the images is unavoidable, then prefer vector image format like SVG. Vector format usually means much smaller file sizes.

I don't know how essential is this screenshot. If it is not essential, just don't add the image. Otherwise, use [Git LFS](https://git-lfs.com/) for more efficient version control of binary files.

Reply:
The image is removed.